### ISSUE TITLE
Links generados en la tarjeta salen con fallo en las URLs

### ISSUE THREAD

**Issue by annabranco** [Adalab/dorcas-s3-ohana-group#23]
La base de las URLs generadas en la tarjeta final sale duplicada.

Por ejemplo:
```
http://github.com/https://github.com/annabranco
http://linkedin.com/in/https://www.linkedin.com/in/annabranco
```

**Comment by oneeyedman** [comment]
Efectivamente, echad un ojo a la documentación de la API, en el ejemplo del json que ha que enviar lo veréis: https://books.adalab.es/materiales-front-end-d/proyectos/p2_anexo

**Comment by annabranco** [comment]
Gracias, @oneeyedman!

**Pues resumiendo:**
De acuerdo con la documentación, los datos de salida no pueden contener los prefixos que hemos añadido a los input, tales como 
`mailto:`, `tel:`, `https://www.linkedin.com/in/`y `http://github.com/`
pues estan informaciones ya son añadidas automáticamente en back por el server de la API.

Los datos tienen que salir así:
![image](https://user-images.githubusercontent.com/39792862/44000824-15b8c7d8-9e27-11e8-9cf1-0bebf9449fef.png)

Tenemos que cambiar los datos de salida de los input y eliminar los prefixos eue hemos añadido. 😉

**Comment by annabranco** [comment]
👏🏼
